### PR TITLE
Document decision for feature to omit RBAC creation on org namespaces

### PR DESCRIPTION
**Pull request by TheBigLee**
The documented decision for the new feature implemented in this PR: https://github.com/appuio/appuio-cloud-agent/pull/54

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

**Review by simu — CHANGES_REQUESTED** [review]
Actual new documentation page is missing.

Also spelling: "ommit" should be "omit"

**Comment by TheBigLee** [comment]
> Actual new documentation page is missing.
> 
> Also spelling: "ommit" should be "omit"

Git is hard :see_no_evil:

Fixed the typo and added the actual document to the PR.

File: docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= Omit creation of default RBAC rules on select organization namespaces
+
+== Problem
+
+The "Managed by VSHN" appcat services provision the instances in a separate namespace that has the same organization label as the namespace where the claim originates from. By default, this will give the organization users full access to those managed namespaces and would allow the users to manipulate the managed appcat services in an undesired way.
+To avoid that, the default RBACs should not be created for those namespaces.
```
Comment (tobru) [inline docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc:6]: I recommend not binding that problem to a particular product. I'd rather state that in a more generic way.

If you want to stick with that: appcat -> "VSHN Application Catalog"

**Review by simu — APPROVED** [review]
LGTM overall

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc (hunk 1 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc]: I have rephrased the paragraph. Does it work for you this way?

Comment (tobru) [reply docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc]: Yes, now it makes most sense to me. Thanks.

**Review by tobru — APPROVED** [review]

**Pull request merged**